src/sentry/utils/auth.py

```diff
@@ -111,18 +112,18 @@ def get_login_redirect(request, default=None):
     if not login_url:
         return default
 
-    if not is_valid_redirect(login_url, host=request.get_host()):
+    if not is_valid_redirect(login_url, allowed_hosts=(request.get_host(),)):
         login_url = default
 
     return login_url
 
 
-def is_valid_redirect(url, host=None):
+def is_valid_redirect(url: str, allowed_hosts: Optional[Container] = None) -> bool:
```

Comment:
Internally, Django sets this as default `set()` if None and just does `in` on it.